cfp/models.py

```diff
@@ -12,6 +17,20 @@ class Proposal(models.Model):
          (6, 'Performant Python'),
          (7, 'Scalable Python'),
          (8, '/etc')])
-
+    speaker_bio = models.TextField(verbose_name="Speaker Bio", max_length=500)
```

Comment:
@OnurOzayyy, you are right and we should limit the speaker_bio candidate talks length. What Daniel is saying is that is preferable to make the limitation in the html form rather than in the backend, as frontend changes are significantly cheaper.